pkg/addons/addons.go

```diff
@@ -578,3 +580,54 @@ func UpdateConfigToDisable(cc *config.ClusterConfig) {
 		}
 	}
 }
+
+// CheckPaused checks whether the cluster is paused before enable/disable an addon.
+func CheckPaused(profile string, enable bool) error {
```

Comment:
The name of the func `CheckPaused` and the logic of it is a bit counter-intuitive in my opinion. For example, if a func was named `CheckRunning` I would assume it to throw an error if it wasn't running, but `CheckPaused` throws an error if it is paused, so the name `CheckUnpaused` would make more sense to me.

ie. Reading through the func makes me think we want the cluster to be paused
```
		if len(args) != 1 {
			exit.Message(reason.Usage, "usage: minikube addons disable ADDON_NAME")
		}
		err := addons.CheckPaused(ClusterFlagValue(), false)
		if err != nil {
			exit.Error(reason.InternalAddonDisablePaused, "disable failed", err)
		}
```

Reply:
@norshtein I agree with spowelljr  I think the name should be something like"verifyNotPaused" or "CheckNotPaused" would be a better name

Reply:
Thanks for the suggestion, that makes great sense. I've renamed the function to `VerifyNotPaused`